core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccess.scala

```diff
@@ -43,8 +43,10 @@ trait DataAccess
       TableQuery[MethodConfigurationTable].delete andThen         // FK to workspace
       TableQuery[EntityTable].delete andThen                      // FK to workspace
       TableQuery[PendingWorkspaceAccessTable].delete andThen      // FK to workspace, user
-      TableQuery[WorkspaceUserShareTable].delete andThen          // FK to workspace, user
-      TableQuery[WorkspaceGroupShareTable].delete andThen         // FK to workspace, group
+      TableQuery[WorkspaceUserShareTable].delete andThen          // FK to workspace, user share
```

Comment:
my intention here was to name the specific tables the FKs reference, so I would like "user" and "group" for these 4.